.travis.yml

```diff
@@ -1,9 +1,8 @@
 sudo: false
 language: node_js
 node_js:
-  - "iojs"
-  - "0.10"
-  - "0.12"
+  - "4.2"
```

Comment:
Good idea, I will try that. It's unclear from the documentation if that's how it resolves. https://docs.travis-ci.com/user/languages/javascript-with-nodejs